templates/appliance/plex.html

```diff
@@ -0,0 +1,171 @@
+{% extends "appliance/_base-appliance.html" %}
+
+{% block title %}Plex Media Server Ubuntu Appliance{% endblock %}
+
+{% block meta_description %}{% endblock %}
```

Comment:
Will this leave the description blank? Maybe we should put something here.

Reply:
Have requested one in the copydoc

Reply:
Thanks but for now let's grab a meaningful paragraph from the description

Reply:
Yeah, will do